contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2764,6 +2764,53 @@ SELECT
 WHERE FALSE;
 GRANT SELECT ON sys.spatial_index_tessellations TO PUBLIC;
 
+CREATE OR REPLACE VIEW sys.asymmetric_keys
+AS
+SELECT 
+    CAST('' as sys.sysname) AS name
+  , CAST(0 as int) AS principal_id
+  , CAST(0 as int) AS asymmetric_key_id
+  , CAST('a' as char(2)) AS pvt_key_encryption_type
+  , CAST('' as sys.nvarchar(60)) AS pvt_key_encryption_type_desc
+  , CAST(null as sys.varbinary(32)) as thumbprint
+  , CAST('a' as char(2)) AS algorithm
```

Comment:
sys.char

Reply:
applicable everywhere

Reply:
Could not apply this as it throws an error.